charts/alfresco-repository/values.yaml

```diff
@@ -214,6 +214,7 @@ ingress:
   enabled: true
   # -- provide annotations for nginx ingress (no toher ingress is supported at
   # that point)
+  className: nginx
```

Comment:
documentation is automatically generated by the precededing `# --` so make sure so move it either after annotations or before the comment

you can add a comment to className as well